backend/libs/room-operations.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { DocumentClient } from 'aws-sdk/clients/dynamodb';
+import { Room } from '../../extension/src/types';
+import { createRoom, joinExistingRoom } from './room-operations';
+
+const isTest = process.env.JEST_WORKER_ID;
+const config = {
+    convertEmptyValues: true,
+    ...(isTest && {
+        endpoint: 'localhost:8000',
+        sslEnabled: false,
+        region: 'local-env',
+    }),
+};
+const ddb = new DocumentClient(config);
+const tableName = 'watch-with-me-dev-RoomTable';
```

Comment:
`'watch-with-me-test-RoomTable'` ?

Reply:
Well as you can see tests are green and even locally using 'test' did not work. Since DDB is emulated locally, I guess this does not really matter does it ?

Reply:
So long as it's dropped after the tests, it's OK :)